src/store/GlobalContext.jsx

```diff
@@ -61,8 +59,10 @@ function GlobalProvider({ children }) {
         handleSearchButton: toggleSearchField,
         searchFieldVisible,
         handleGeolocate,
-        weatherObj: weatherObj,
+        weatherData: weatherData,
```

Comment:
you may collapse it to just weatherData,

like you did to handleGeolocate

Reply:
fair